wrappers/golang/core/vec_ops_test.go

```diff
@@ -0,0 +1,23 @@
+package core
+
+import (
+	"github.com/ingonyama-zk/icicle/wrappers/golang/cuda_runtime"
```

Comment:
Use the same import name here and in `wrappers/golang/core/vec_ops.go`

```suggestion
	cr "github.com/ingonyama-zk/icicle/wrappers/golang/cuda_runtime"
```